tests/kafkatest/tests/core/transactions_mixed_versions_test.py

```diff
@@ -198,7 +198,7 @@ def test_transactions_mixed_versions(self, old_kafka_version, metadata_quorum=qu
         self.kafka.interbroker_security_protocol = security_protocol
         self.kafka.logs["kafka_data_1"]["collect_default"] = True
         self.kafka.logs["kafka_data_2"]["collect_default"] = True
-        self.kafka.logs["kafka_operational_logs_debug"]["collect_default"] = True
+        self.kafka.logs["kafka_operational_logs_debug"]["collect_default"] = if old_kafka_version == str(LATEST_3_2) False else True
```

Comment:
It seems 3.1.2 has same issue. see https://github.com/apache/kafka/blob/3.1.2/metadata/src/main/java/org/apache/kafka/metadata/BrokerRegistration.java

Maybe `LATEST_3_1` should be included too?